picard/script/functions.py

```diff
@@ -870,14 +870,14 @@ def func_truncate(parser, text, length):
 
 
 @script_function(check_argcount=False, documentation=N_(
-    """`$swapprefix(text,*prefixes="a","the")`
+    """`$swapprefix(text,prefix...)`
 
 Moves the specified `prefixes` from the beginning to the end of `text`.
```

Comment:
Since function description is modified, text should be modified accordingly (`prefixes` vs `prefix`)

Reply:
IMO no. I know this has dual use - user documentation and technical documentation - but the code is simple and `prefixes` is the correct variable name, and I did not want to make code changes in this PR.

But if the consensus is that we should do this, I can make this change.

Reply:
I see how `prefix` makes sense here. I would add a sentence like "Multiple prefixes can be specified." or such to make this even more clear.

Reply:
Or put `$swapprefix(text,prefix1,...)`